src/backend/columnar/columnar_tableam.c

```diff
@@ -638,6 +642,8 @@ static bool
 columnar_tuple_satisfies_snapshot(Relation rel, TupleTableSlot *slot,
 								  Snapshot snapshot)
 {
+	CheckCitusVersion(ERROR);
+
```

Comment:
As far as I know, we already don't support any features that need to call this callback, so maybe we should have just thrown an error for that.

That is why codecov complains we don't test this change.